sdk/examples/wallet/01_create_wallet.rs

```diff
@@ -2,9 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 
 //! In this example we will create a new wallet.
-//! Rename `.env.example` to `.env` first.
 //!
-//! `cargo run --example create_wallet --release`
+//! Make sure there's no `example.stronghold` file and no `example.walletdb` folder yet!
+//!
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --all-features --example create_wallet
```

Comment:
Don't know if we should suggest all feature for every example, maybe be way longer than necessary...